content/docs/use-cases/versioning-data-and-model-files/tutorial.md

```diff
@@ -317,26 +317,24 @@ When you have a script that takes some data as an input and produces other data
 > ```
 
 ```dvc
-$ dvc run -f Dvcfile \
-          -d train.py -d data \
-          -M metrics.csv \
-          -o model.h5 -o bottleneck_features_train.npy -o bottleneck_features_validation.npy \
+$ dvc run -n train -d train.py -d data \
+          -o model.h5 -o bottleneck_features_train.npy \
+          -o bottleneck_features_validation.npy -M metrics.csv \
           python train.py
 ```
 
-Similar to `dvc add`, `dvc run` creates a
-[DVC-file](/doc/user-guide/dvc-files-and-directories) named `Dvcfile` (specified
-using the `-f` option). It tracks all outputs (`-o`) the same way as `dvc add`
-does. Unlike `dvc add`, `dvc run` also tracks dependencies (`-d`) and the
-command (`python train.py`) that was run to produce the result. We call such a
-DVC-file a "stage file".
+`dvc run` creates a pipeline stage named `train` (specified using the `-n`
+option) in [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file)
+file. It tracks all outputs (`-o`) the same way as `dvc add` does. Unlike
```

Comment:
```suggestion
`dvc run` writes a pipeline stage named `train` (specified using the `-n`
option) in [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file).
It tracks all outputs (`-o`) the same way as `dvc add` does. Unlike
```

Reply:
![run](https://user-images.githubusercontent.com/46163196/86557482-e4439500-bf73-11ea-931e-02c2bac8dfd6.PNG)

Do we need to change it from here also?

Reply:
Yes but it's out of scope. The whole ref. needs to be rewritten soon. I'm committing my suggestion above to save time. Please just fix the formatting. Resolving here.

Reply:
Actually the command was already rewritten so yes, just this intro to that command should be rephrased... But still, out of scope for this PR. I'll send a separate one.

Reply:
#1537